spec/interface/belongs_to_polymorphic_spec.rb

```diff
@@ -15,47 +15,51 @@
     create_table :villains
   end
 
-  specify { StoryArc::POLYMORPHEUS_ASSOCIATIONS.should == %w[hero villain] }
-  specify { Superpower::POLYMORPHEUS_ASSOCIATIONS.should == %w[superhero
-                                                               supervillain] }
+  specify do
+    expect(StoryArc::POLYMORPHEUS_ASSOCIATIONS).to eq(%w[hero villain])
+  end
+  specify do
+    expect(Superpower::POLYMORPHEUS_ASSOCIATIONS).to eq(%w[superhero supervillain])
+  end
```

Comment:
Should these changes be in the previous commit?

Reply:
@bryan-ash I don't think so. The previous commit dealt with uses of `its`, but still remains entirely compatible with rspec v2. This commit makes all of the changes required to switch to v3.

Reply:
@bryan-ash Ah, I see what you mean though. The previous commit _did_ switch to some uses of the `expect` syntax. I'd forgotten about that.